sample_apps_reinvent2021/python/utils/TimestreamDependencyHelper.py

```diff
@@ -218,17 +228,15 @@ def get_service_name(self, stage, region):
         else:
             return "{}.{}.timestream.aws.internal".format(region, stage)
 
-    def create_s3_bucket(self, bucket_name, region=None):
+    def create_s3_bucket(self, bucket_name):
         print("\nCreating S3 bucket")
         try:
-            if region:
-                s3_client = boto3.client('s3', region_name=region)
-                location = {'LocationConstraint': region}
-                s3_client.create_bucket(Bucket=bucket_name,
-                                        CreateBucketConfiguration=location)
-            else:  # default region is us-east-1
-                s3_client = boto3.client('s3')
-                s3_client.create_bucket(Bucket=bucket_name)
+            if self.region == 'us-east-1':
+                self.s3_client.create_bucket(Bucket=bucket_name)
+            else:
+                self.s3_client.create_bucket(Bucket=bucket_name,
```

Comment:
You don't need lines 234- 236, You can always specify location constraint for all regions

Reply:
I didn't have a chance to look into why, but this is here because passing `us-east-1` (the default S3 region) as a location constraint resulted in either an initialization error or a bucket creation error.